x-pack/legacy/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -8,7 +8,7 @@ import path from 'path';
 
 export const paths = {
   archivesPath: path.resolve(__dirname, '../../../.chromium'),
-  baseUrl: 'https://s3.amazonaws.com/headless-shell/',
+  baseUrl: 'https://storage.googleapis.com/headless_shell/',
```

Comment:
would it be overkill to make this configurable?

Reply:
I'm leaning towards yes: the checksums make it impossible to implement this yourself (in the case of our users), and having different vendors for distributing this means having to maintain multiple places + another layer of debugging for services in case someone does something goofy. Honestly I don't envision this changing much so I'm ok with making it an application artifact versus a configuration artifact.

Reply:
Looks like we understand now that the baseUrl is only a factor in a dev and build environment :D